Commons/src/main/java/eu/neclab/ngsildbroker/commons/controllers/QueryControllerFunctions.java

```diff
@@ -58,7 +58,7 @@ public static ResponseEntity<String> getEntity(EntryQueryService queryService, H
 			return ResponseEntity.status(HttpStatus.BAD_REQUEST).header(HttpHeaders.CONTENT_TYPE, AppConstants.NGB_APPLICATION_JSON)
 					.body(new RestResponse(ErrorType.BadRequestData, "id is not a URI").toJson());
 		}
-		String originalQuery = NGSIConstants.QUERY_PARAMETER_ID + "=" + entityId;
+		String originalQuery = entityId;
```

Comment:
Please test properly and add this test cases in FT sheet.

Reply:
Done